keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationEventTests.kt

```diff
@@ -38,7 +38,7 @@ internal class ApplicationEventTests {
 
     publisher.publishEvent(TestEvent(this))
 
-    val eventThread = listener.awaitInvoked(Duration.ofMillis(100))
+    val eventThread = listener.awaitInvoked(Duration.ofMillis(300))
```

Comment:
It was timing out before the event got received. maybe github actions was being weird? So I bumped this up a little bit because 300 millis is not that much and I prefer it to the whole build failing

Reply:
That's just the max time it will wait, right? It's not a `sleep` call.

Reply:
ya